.github/workflows/testbench.yml

```diff
@@ -47,15 +47,20 @@ jobs:
 
       # testbench needs some topologies.
       - name: build test topologies
+        continue-on-error: true
         run: CMAKE_BUILD_TYPE=Release ./scripts/docker-run.sh
-          ./scripts/build-tools.sh -Y -A ||
+          ./scripts/build-tools.sh -A -Y ||
```

Comment:
@cgturner1 Right, so with this workflow, the workspace layout is different (than with zephyr.yml checkous), so the topologu cmake rules won't find the built alsa tools, and you get an error about "unknown target 'topologies1'"

But, but, I guess the easy fix is to remove docker-run.sh from here (L51) altogether. Now that we don't need the alsa tools from docker, we should just be able to run build-tools.sh directly in the Ubuntu environment.